ts/components/conversation/ConversationHeader.tsx

```diff
@@ -377,7 +380,7 @@ export class ConversationHeader extends React.Component<Props> {
           <SessionIconButton
             iconType={SessionIconType.Exit}
             iconSize={SessionIconSize.Medium}
-            onClick={onCloseOverlay}
+            onClick={onResetSession}
```

Comment:
This name is a bit confusing. `onResetSession` could be confused for the Signal session.